src/hooks/useNavigationEffect.js

```diff
@@ -0,0 +1,52 @@
+import { useEffect } from "react";
+
+import { MOUSE_EVENTS, KEYS, NON_NAVIGATION_KEYS } from "src/constants";
+
+export const useNavigationEffect = () => {
+  useEffect(() => {
+    if (typeof window !== "undefined" && typeof document !== "undefined") {
+      const eventListener = event =>
+        Object.values(KEYS).includes(event.key) &&
+        !NON_NAVIGATION_KEYS.includes(event.key) &&
+        togglePointerEvents(MOUSE_EVENTS.KEY_PRESS);
+
+      window.addEventListener("keydown", eventListener);
+      return () => window.removeEventListener("keydown", eventListener);
```

Comment:
Do you think is a good idea to disable the eventListener of the mousemove here? I was in doubt if I put it or not

Reply:
Yes, it's always better.

The callback function you return gets called automatically when the component unmounts, which is what we want.

If it's not there, we could end up with multiple active event listeners.